xc/xc7/tests/soc/ibex/CMakeLists.txt

```diff
@@ -1,78 +1,138 @@
-add_file_target(FILE led.vmem)
-add_file_target(FILE pins_artya7.sdc)
+get_filename_component(IBEX_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../../../../../third_party/ibex ABSOLUTE)
+
+set(SV2V_ROM_INIT_FILE led.vmem)
+set(SV2V_IBEX_FLAGS --define=RegFile=ibex_pkg::RegFileFF --define=SRAMInitFile=${SV2V_ROM_INIT_FILE})
+
+set(SOURCE_FILE_LIST
```

Comment:
Can `fusesoc` supply this list of files during CMake configure time?

Reply:
We have to generate all the sources to obtain this list.

Reply:
I created  a `generate.sh` script which can be used for obtaining the dependency list. During the CMake configuration, the script copies the files to tmp directory, applies the patches and generates the sources. Then the `fusesoc_get_sources.py` is used to obtain the list of source files.